app/controllers/autocomplete_controller.rb

```diff
@@ -169,7 +169,7 @@ def potential_matches(return_requests=true)
     pmatches = return_requests ?
       signup.offer_potential_matches.sort.reverse.map {|pm| pm.request_signup.pseud.byline} :
       signup.request_potential_matches.sort.reverse.map {|pm| pm.offer_signup.pseud.byline}
-    pmatches.select! {|pm| pm.match(/#{search_param}/)} if search_param.present?
+    pmatches.select! {|pm| pm.match(/#{ Regexp.escape(search_param) }/)} if search_param.present?
```

Comment:
Similarly, to fix the Hound violations, I'd suggest changing this to:
```suggestion
    pmatches.select! { |pm| pm.match(/#{Regexp.escape(search_param)}/) } if search_param.present?
```